v2/schema/context.jsonld

```diff
@@ -0,0 +1,112 @@
+{
+  "@context": {
+    "dcat": "http://www.w3.org/ns/dcat#",
+    "@vocab": "http://www.w3.org/ns/dcat#",
+    "dc": "http://purl.org/dc/terms/",
+    "dctypes": "http://purl.org/dc/dcmitype/",
```

Comment:
I was thinking the same when translating between the existing schema and this one. It is confusing, and I don't recall what it's all about. However, if you look at the dcat vocab spec, it uses dcterms for everything. https://www.w3.org/TR/vocab-dcat/